opennms-webapp/src/main/java/org/opennms/web/filter/EqualsFilterNullAware.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2007-2014 The OpenNMS Group, Inc.
```

Comment:
Use 
  Copyright (C) 2002-2020 The OpenNMS Group, Inc.
  OpenNMS(R) is Copyright (C) 1999-2020 The OpenNMS Group, Inc.
for new files.

Reply:
:+1: fixed